tests/run_tests.sh

```diff
@@ -5,20 +5,21 @@ function show_help_and_exit()
     echo "Usage ${SCRIPT} [options]"
     echo "    options with (*) must be provided"
     echo "    -h -?          : get this help"
-    echo "    -c             : specify test cases to execute (default: none, executed all matched)"
-    echo "    -d             : specify DUT name (*)"
-    echo "    -e             : specify extra parameter(s) (default: none)"
-    echo "    -f             : specify testbed file (default testbed.csv)"
-    echo "    -i             : specify inventory name"
-    echo "    -k             : specify file log level: error|warning|info|debug (default debug)"
-    echo "    -l             : specify cli log level: error|warning|info|debug (default warning)"
-    echo "    -m             : specify test method group|individual|debug (default group)"
-    echo "    -n             : specify testbed name (*)"
+    echo "    -c <testcases> : specify test cases to execute (default: none, executed all matched)"
+    echo "    -d <dut name>  : specify DUT name (*)"
+    echo "    -e <paramters> : specify extra parameter(s) (default: none)"
+    echo "    -f <tb file>   : specify testbed file (default testbed.csv)"
+    echo "    -i <inventory> : specify inventory name"
+    echo "    -k <log level> : specify file log level: error|warning|info|debug (default debug)"
+    echo "    -l <log level> : specify cli log level: error|warning|info|debug (default warning)"
```

Comment:
Should we mention log vs. cli log level in the tag so it's more clear which is which at a glance?

Reply:
Thanks! Fixed.